Kernel/Syscalls/futex.cpp

```diff
@@ -105,9 +105,7 @@ ErrorOr<FlatPtr> Process::sys$futex(Userspace<Syscall::SC_futex_params const*> u
 
     switch (cmd) {
     case FUTEX_WAIT:
-    case FUTEX_WAIT_BITSET:
-    case FUTEX_REQUEUE:
-    case FUTEX_CMP_REQUEUE: {
+    case FUTEX_WAIT_BITSET: {
```

Comment:
The closing brace for this appears to be on line 122/124, after the case for WAKE_BITSET. which, it should be just after the break on line 119/117. was this pre-existing?

Reply:
or wait what's going on here. why are there two cases for FUTEX_WAIT_BITSET? this is some funky compiler abuse.

Reply:
That was pre-existing. And kinda horrible...

Reply:
Actually looking again, it's even worse - there's a block of code that's expressed both in a case statement and an if statement! eugh this is nasty...

Reply:
To further make this weirder, the LibC side of this is expressed as a list of calls which do not use timeout, whereas the kernel side is a list of calls which do...